src/dev/get_angular_expression_ast.js

```diff
@@ -0,0 +1,684 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
__note:__ I'm not sure that we can just simply pull in the part of angular without any license notice/attribution and such, what'd would be the process in case we decide to do something like this @epixa?

__question:__ @LeanidShutau could you please provide us with more insight on that? Why do we want to bring in part of the Angular internals? Do we have any other more maintainable or simpler alternatives? How difficult it is to write our own babel-based super-simplified parser that is specifically tailored for our use case?

Reply:
@azasypkin Done. Angular internal parser has been replaced by simple solution limiting allowed syntax to `<JS code> | i18n: <JS code>` and using `@babel/parser` to parse JS. Other syntax is redundant for angular translation expressions.